android/app/src/googleplay/java/org/fossasia/openevent/utils/MicrolocationClusterWrapper.java

```diff
@@ -0,0 +1,29 @@
+package org.fossasia.openevent.utils;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.clustering.ClusterItem;
+
+import org.fossasia.openevent.data.Microlocation;
+
+/**
+ * Created by rohanagarwal94 on 6/7/17.
+ */
+public class MicrolocationClusterWrapper implements ClusterItem {
```

Comment:
Ok, then move it to some specific package, utils package should be for app scope components